test/visual/align-themes.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+
+<head lang="en">
+  <meta charset="UTF-8">
+  <title>Drop down menu tests</title>
+  <script src="../../../webcomponentsjs/webcomponents-lite.js"></script>
+  <link rel="import" href="../../../polymer/polymer.html">
+  <script>
+    // eslint-disable-next-line no-unused-vars
+    /* global theme */
+    const dir = window.location.search.replace(/.*dir=(\w+).*/, '$1') || 'ltr';
+    document.documentElement.setAttribute('dir', dir);
+
+    const theme = 'lumo';
+    document.write(`<link rel="import" href="../../theme/${theme}/vaadin-select.html">`);
+    document.write(`<link rel="import" href="../../../vaadin-list-box/theme/${theme}/vaadin-list-box.html">`);
+    document.write(`<link rel="import" href="../../../vaadin-item/theme/${theme}/vaadin-item.html">`);
+  </script>
+</head>
+
+<body>
+  <style>
+    body {
+      font-family: sans-serif;
+    }
+
+    .capture-block {
+      display: inline-block;
+      width: 800px;
+      height: 350px;
+    }
+  </style>
+
+  <div class="capture-block" id="select">
+
+    <vaadin-select label="Selected value" value="Selected" id="general" theme="align-left">
```

Comment:
Two final notes (apologise for this many review comments submitted separately): 

1. let's update `label` for these selects to use `left`, `center` and `right` so it would be self-descriptive

2. let's not use "Very very long item label" below, as we are testing `align-center` theme and that is unclear with the long item IMO

Reply:
No probs.

1. Updated to reflect the theme name.

1. It was there to check that there is no mask image on sides, updated now.